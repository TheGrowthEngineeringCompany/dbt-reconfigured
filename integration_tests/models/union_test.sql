{{
    config(
        materialized="table",
        schema="test_data"
    )
}}

{{ reconfigured.union_with_id_prefix(
    id_cols=['id', 'another_id'],
    relations=[
      [ref('union_source_1'), 'Source1:'],
      [ref('union_source_2'), 'Source2:'],
    ]
) }}
