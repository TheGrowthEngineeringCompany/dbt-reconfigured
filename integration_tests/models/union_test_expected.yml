version: 2

models:
  - name: union_test_expected
    tests:
      - dbt_utils.equality:
          compare_model: ref('union_test')
          compare_columns:
            - id
            - another_id
            - value_1
            - value_2
